tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -1866,5 +1866,11 @@ Not everything needs to be implemented for ocis. While the oc10 testsuite covers
 - [apiShareManagementBasicToShares/createShareToSharesFolder.feature:131](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareManagementBasicToShares/createShareToSharesFolder.feature#L131)
 - [apiShareManagementBasicToShares/createShareToSharesFolder.feature:132](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareManagementBasicToShares/createShareToSharesFolder.feature#L132)
 
+#### [resource path is no longer included in the returned error message](https://github.com/owncloud/ocis/issues/3344)
```

Comment:
Note: I reopened issue #3344 and made a comment there. QA will look at the failures and sort out what to do next.